dev/group_vars/apim.yml

```diff
@@ -22,6 +22,14 @@ target: /mnt
 product_package_location: files
 #jdbc_driver: mysql-connector-java-5.1.47-bin.jar
 
+# Performance tuning configurations
+enable_performance_tuning: true
```

Comment:
Please keep the default as false

Reply:
Fixed https://github.com/wso2/ansible-apim/pull/45/commits/48626889a1dd269050e9be7a121221d2944181d1